cardano-api/src/Cardano/Api/Fees.hs

```diff
@@ -797,6 +797,10 @@ data TxBodyErrorAutoBalance =
        -- word parameter, for eras that use this parameter.
      | TxBodyErrorMissingParamCostPerWord
 
+       -- | The 'ProtocolParameters' must provide the value for the cost per
+       -- byte parameter, for eras that use this parameter.
+     | TxBodyErrorMissingParamCostPerByte
+
```

Comment:
Both `TxBodyErrorMissingParamCostPerWord` and `TxBodyErrorMissingParamCostPerByte` appear to be unused.  Should I delete @Jimbo4350?  Or is a change needed to ensure they are used?

Reply:
Deleted